deb/publish/publish.sh

```diff
@@ -12,9 +12,6 @@ set -euxo pipefail
 # $$ Contains current pid
 D="$AGENT_WORKDIR/$$"
 
-# Convert string to array to correctly escape cli parameter
-SSH_OPTS=($SSH_OPTS)
```

Comment:
> Feel free to comment / review even after the PR merge.

@dduportal I think this was incomplete. The variable is still used at the bottom of this file:

```
deb/publish/publish.sh
113:            -e "ssh ${SSH_OPTS[*]}" \
138:            -e "ssh ${SSH_OPTS[*]}" \
147:    echo "SSH_OPTS: ${SSH_OPTS[*]}"
```